Python_DataEngineering/DDL_Statements.txt

```diff
@@ -0,0 +1,13 @@
+Query Statements Used todo the assigned Task:
+---------------------------------------------------
+
+"create table Bike_Detail(bikepointname varchar(255),noofbike varchar(255),datetime varchar(255))"
+"create table time_snap_shot_detail(datetime varchar(255),noofbike varchar(255))"

```

Comment:
The intent was to have 2 tables:

- One table to have only the realtime bike availability 
- One to have historical availability for all bike points, along with the timestamps of when that data was recorded

Can you modify your schema accordingly?

Reply:
So, here the first tables does what you have asked for . The Table Bike Detail stores the history data of no of bikes available in all the bike points when it was recorded.  I will do change my schema for the 1st point you have asked for.